ocaml/tests/suite_alcotest.ml

```diff
@@ -71,7 +71,6 @@ let () =
       @ Test_vm.tests
       @ Test_cpuid_helpers.tests
       @ Test_pool_cpuinfo.tests
-      @ Test_dbsync_master.tests
```

Comment:
this made me suspicious, but it looks like the dbsync master tests were actually only testing the tools sr :heavy_check_mark: